x-pack/test/screenshot_creation/apps/ml_docs/anomaly_detection/generate_anomaly_alerts.ts

```diff
@@ -0,0 +1,139 @@
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+import { ECOMMERCE_INDEX_PATTERN } from '..';
+
+function createTestJobAndDatafeed() {
+  const timestamp = Date.now();
+  const jobId = `high_sum_total_sales_${timestamp}`;
+
+  return {
+    job: {
+      job_id: jobId,
+      description: 'test_job',
+      groups: ['ecommerce'],
+      analysis_config: {
+        bucket_span: '1h',
+        detectors: [
+          {
+            detector_description: 'High total sales',
+            function: 'high_sum',
+            field_name: 'taxful_total_price',
+            over_field_name: 'customer_full_name.keyword',
+            detector_index: 0,
+          },
+        ],
+        influencers: ['customer_full_name.keyword', 'category.keyword'],
+      },
+      data_description: {
+        time_field: 'order_date',
+        time_format: 'epoch_ms',
+      },
+      analysis_limits: {
+        model_memory_limit: '13mb',
+        categorization_examples_limit: 4,
+      },
+    },
+    datafeed: {
+      datafeed_id: `datafeed-${jobId}`,
+      job_id: jobId,
+      query: {
+        bool: {
+          must: [
+            {
+              match_all: {},
+            },
+          ],
+          filter: [],
+          must_not: [],
+        },
+      },
+      query_delay: '120s',
+      indices: [ECOMMERCE_INDEX_PATTERN],
+    } as unknown as estypes.MlDatafeed,
+  };
+}
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+  const pageObjects = getPageObjects(['triggersActionsUI']);
+  const commonScreenshots = getService('commonScreenshots');
+  const browser = getService('browser');
+
+  const screenshotDirectories = ['ml_docs', 'anomaly_detection'];
+
+  let testJobId = '';
+
+  describe('anomaly detection alert', function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/ecommerce');
+      await ml.testResources.createIndexPatternIfNeeded('ft_ecommerce', 'order_date');
+
+      const { job, datafeed } = createTestJobAndDatafeed();
+
+      testJobId = job.job_id;
+
+      // Set up jobs
+      // @ts-expect-error not full interface
+      await ml.api.createAnomalyDetectionJob(job);
+      await ml.api.openAnomalyDetectionJob(job.job_id);
+      await ml.api.createDatafeed(datafeed);
+      await ml.api.startDatafeed(datafeed.datafeed_id);
+      await ml.api.waitForDatafeedState(datafeed.datafeed_id, DATAFEED_STATE.STARTED);
+      await ml.api.assertJobResultsExist(job.job_id);
+    });
+
+    after(async () => {
+      await ml.api.deleteAnomalyDetectionJobES(testJobId);
+      await ml.api.cleanMlIndices();
+      await ml.alerting.cleanAnomalyDetectionRules();
+    });
+
+    describe.only('overview page alert flyout controls', () => {
+      it('can create an anomaly detection alert', async () => {
```

Comment:
We usually use this to describe the screenshot(s) we're taking. I'd suggest something like this:
```suggestion
      it('alert flyout screenshots', async () => {
```